src/NHibernate/Impl/SessionFactoryImpl.cs

```diff
@@ -1460,7 +1461,7 @@ public virtual ISession OpenSession()
 				if (_interceptor != null)
 				{
 					// NH specific feature
-					// _interceptor may be the shared accros threads EmptyInterceptor.Instance, but that is
+					// _interceptor may be the shared across threads EmptyInterceptor.Instance, but that is
```

Comment:
This comment is now violated. With this PR, it may be another shared interceptor than the empty one which does nothing of the session and so is harmless.

This introduces threading bugs. This is a no go.